sdk/rust/oak/src/lib.rs

```diff
@@ -462,3 +461,69 @@ pub fn run_event_loop<T: crate::io::Decodable, N: Node<T>>(mut node: N, in_handl
         }
     }
 }
+
+/// Register a new node entrypoint.
+///
+/// This registers the entrypoint name and the expression used to construct the
+/// node instance.
+///
+/// ```
+/// #[derive(Default)]
+/// struct DummyNode;
+///
+/// impl oak::grpc::OakNode for DummyNode {
```

Comment:
I think it would be cleaner (and more accurate) to `impl oak::Node` instead.

Reply:
Done! The amount of hidden code in the example is now quite large, do you have any tips on how we could potentially slim that down a bit?